Documentation/ansible-install.md

```diff
@@ -0,0 +1,59 @@
+# Installation using openshift-ansible
+
+Using ansible is the recommend installation method for installing Metering on to an Openshift cluster.
+
+The [openshift-metering playbook][metering-playbook] is located within the [openshift-ansible repo][openshift-ansible].
+
+At the time this document was written, the openshift-metering playbook is available in the master branch and should be in the release-3.11 branch when it's created as well.
+
+## Configuration
+
+All of the supported configuration options are documented in [configuring metering][configuring-metering].
+To supply custom configuration options set the `openshift_metering_config` variable to a dictionary containing the contents of the `Metering` `spec` field you wish to set.
+
+For example:
+
+```
+openshift_metering_config:
+  metering-operator:
+    config:
+      awsAccessKeyID: "REPLACEME"
+```
+
+## Install
+
+First, clone openshift-ansible:
+
+```
+git clone https://github.com/openshift/openshift-ansible
+```
+
+Next, set the `openshift_metering_install` variable to true:
+
+```yaml
+openshift_metering_install: true
+```
+
+Finally, run the playbook:
+
+```
+ansible-playbook playbooks/openshift-metering/config.yml
+```
+
+If you're on GCP use the following instead:
+
+```bash
+ansible-playbook playbooks/openshift-metering/gcp-config.yml
```

Comment:
@chancez, would be helpful for newbies like me to list out the expected pods to be found after a successful installation.